topics/transcriptomics/tutorials/scrna-preprocessing-tenx/tutorial.md

```diff
@@ -0,0 +1,449 @@
+>
+> > ### {% icon solution %} Solution
+> > 1. We see the blue knee line cross the threshold of barcodes at just greater than the 100 Rank on the horizontal log scale, so we can assume that our assessment that there should be around 300 cells is correct.
+> > 1. The threshold is given by the inflection line which tells us that any cell with a total UMI count less than 3 should be considered empty.
+> >   **However**, this dataset has been greatly sub-sampled for training purposes (~1000 down to ~300 cells) so the sensitivity of this value should be taken into account. Due to the already low number of cells, we should realistically set this to 1.
+> {: .solution}
+{: .question}
+
+On large 10x datasets we can use these thresholds as metrics to utilise in our own custom filtering, which is once again provided by the **DropletUtils** tool.
+
+> ### {% icon hands_on %} Hands-on: Custom Filtering
+>
+> **DropletUtils** {% icon tool %} with the following parameters:
+>    - *"Format for the input matrix"*: `Bundled (barcodes.tsv, genes.tsv, matrix.mtx)`
+>        - {% icon param-file %} *"Count Data"*: `output_matrix` (output of **RNA STARsolo** {% icon tool %})
+>        - {% icon param-file %} *"Genes List"*: `output_genes` (output of **RNA STARsolo** {% icon tool %})
+>        - {% icon param-file %} *"Barcodes List"*: `output_barcodes` (output of **RNA STARsolo** {% icon tool %})
+>    - *"Operation"*: `Filter for Barcodes`
+>        - *"Method"*: `EmptyDrops`
+>            - *"Lower-bound Threshold"*: `1`
+>            - *"FDR Threshold"*: `0`
+>        - *"Format for output matrices"*: `Tabular`
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > We set the lower-bound to 1 instead of 3 due to the sparsity of the dataset. On a full dataset, you would insert the value you discovered using the Rank method.
+>    >
+>    > The False Discovery Rate (FDR) is set to zero because the ambient profile is too weak to measure against on this dataset.
+>    >
+>    {: .comment}
+>
+{: .hands_on}
+
+![cells]({{ site.baseurl }}{% link topics/transcriptomics/images/tenx_cells.png %} "Detected Cells (red)")
+
+
+Here we recover 137 high quality cells instead of the detected 272 given previously. On large datasets, this difference can help clean downstream clustering which may appear softer and less well-defined on a more inclusive dataset than that which has undergone tighter quality controls.
+
+# Conclusion
+{:.no_toc}
+
+In this workflow we have learned to quickly perform mapping and quantification of scRNA-seq FASTQ data in a single step via **RNA STARsolo**, and have reproduced a CellRanger workflow using the **DropletUtils** suite, where we further explored the use of barcode rankings to determine better filtering thresholds to generate a high quality count matrix.
+
+A full pipeline which produces both an hda5 and tabular file for inspection is provided [here](workflows/scrna_tenx.ga).
```

Comment:
hda5? Please explain what that is ...
one of your questions is ` "What is an HDF5 file, and why is it important?"` but this part is missing.